mocks/pkg/fixtures/ExpecterTest.go

```diff
@@ -1,18 +1,26 @@
-// Code generated by mockery. DO NOT EDIT.
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
This looks like you used the wrong version to generate the mocks...strange.

Reply:
Agreed, could you verify what mockery you're using to build the mocks? These version strings should be disabled so this is telling me that your mockery probably wasn't reading the config properly. (maybe we should re-introduce the version numbers?)

Reply:
I hadn't used the Makefile, fixed now